src/Components/Settings/SettingsButton.css

```diff
@@ -0,0 +1,6 @@
+#settingsButton {
+	width: 3.125vw;
+	height: 3.125vw;
+	background-color: #778ca1;
+	border: 1px solid #101010;
+}
```

Comment:
- please change 3.125vw to `calc(50vh/9)`
 - please add actionsbutton to the upperrightcontainer, and define the margin as 0.5 of the height unit vlaue.
 - recommended to use a css variable of height-unit for this.

Reply:
I'm having issues converting the sizing and padding of the Settings Button to %s.